hack/quick-install.sh

```diff
@@ -5,7 +5,16 @@
 set -o errexit
 set -o xtrace
 
-bash hack/install-database.sh
+kubectl apply -f - <<< $(helm template kubearchive charts/kubearchive \
+  -s templates/database/database.yaml \
+  --set "database.enabled=true"  --set "global.production=true")
```

Comment:
This will fail to install, and we would be installing KubeArchive two times. I don't think we need to align testing with development as well. With some of the changes proposed here we are already approaching development to testing.

Reply:
Note that with `-s templates/database/database.yaml` it should only render that specific file, so we don't need `ko` to be applied there because we are using the `postgres` image.

I tried the script `hack/quick-install.sh` with these changes and it actually works, but for this specific line you could just try locally:

```bash
 helm template kubearchive charts/kubearchive \
  -s templates/database/database.yaml \
  --set "database.enabled=true"  --set "global.production=true"
  ```

Reply:
> I don't think we need to align testing with development as well. With some of the changes proposed here we are already approaching development to testing.

About this, I think that the sentence I wrote in the Notes for reviewers ("My intention is to move the development closer to the production way and not the other way around.") is a little missleading.

I mean that with this PR, as I am "merging" the helm templates of the database and the script for the database into one, as they are not identical I chose the especifics of the testing environment (script) instead of the development environment (chart templates).

One of the examples of this is that I chose to use the `root` credentials instead of the kubearchive credentials in the `Deployment`

So, I don't think that there is extra work to be done related to the testing/development alignment apart from what I am covering in the current PR

Reply:
> Note that with -s templates/database/database.yaml it should only render that specific file, so we don't need ko to be applied there because we are using the postgres image.

Ah, I need to read better. Sorry about that.